src/mailadm/bot.py

```diff
@@ -34,15 +34,28 @@ def __init__(self, db: DB, account: deltachat.Account):
         self.account = account
         with self.db.read_connection() as conn:
             config = conn.config
-            self.admingrpid = config.admingrpid
+            self.admingrpid = int(config.admingrpid)
 
     @account_hookimpl
     def ac_incoming_message(self, message: deltachat.Message):
         arguments = message.text.split(" ")
-        print("process_incoming command:", arguments)
+        print("process_incoming message:", message.text)
         if not self.check_privileges(message):
-            message.create_chat()
-            message.chat.send_text("Sorry, I only take commands from the admin group.")
+            if message.text[0] == "/":
+                message.create_chat()
+                message.chat.send_text("Sorry, I only take commands from the admin group.")
+            else:
+                admingroup = self.account.get_chat_by_id(self.admingrpid)
+                admingroup.send_text("message by %s:\n%s" %
+                                     (message.get_sender_contact().addr, message.text))
+            return
+
+        if message.quote.get_sender_contact().addr == self.account.get_config("addr"):
```

Comment:
wouldn't this blow up if `quote is None`?

Reply:
oops. fixed